R/drive_publish.R

```diff
@@ -1,9 +1,15 @@
-#' Publish Google Drive file.
+#' Publish Google Drive file
```

Comment:
I think we've decided to stop saying "Google Drive" and, even, "Drive" so much, right? I think we should qualify only in very introductory materials or if we need to explicitly distinguish Drive files from local ones.